cmake/profiles/develop.cmake

```diff
@@ -56,12 +60,21 @@ elseif(MBED_TOOLCHAIN STREQUAL "ARM")
         "-std=gnu++14"
         "-fno-rtti"
         "-fno-c++-static-destructors"
+        "-Os"
     )
     target_compile_options(mbed-os
         PUBLIC
             $<$<COMPILE_LANGUAGE:CXX>:${cxx_compile_options}>
     )
 
+    list(APPEND asm_compile_options
+        "-Os"
```

Comment:
Does passing `-Os` to the assembler perform any optimizations?

Reply:
fixed, removed these from asm